gatsby-config.js

```diff
@@ -6,6 +6,10 @@ require('dotenv').config({
 })
 
 const site_url = 'https://deriv.com'
+const strapi_url = 'https://deriv-com-content.herokuapp.com'
```

Comment:
Let's get all these strapi details from .env file

Reply:
please check my previous commits, I did it as you said, but we have a Vercel build error because we don't have these variables in our Vercel, we need to add all these variables to Vercel environment variables list 🙏